src/dispatch/common.rs

```diff
@@ -77,6 +77,11 @@ async fn check_permissions_and_cooldown_single<'a, U, E>(
     ctx: crate::Context<'a, U, E>,
     cmd: &'a crate::Command<U, E>,
 ) -> Result<(), crate::FrameworkError<'a, U, E>> {
+    // Skip command checks if `FrameworkOptions::skip_checks_for_owners` is set to true
+    if ctx.framework().options.skip_checks_for_owners {
+        return Ok(());
+    }
```

Comment:
Wait.. this skips checks unconditionally for everyone if skip_checks_for_owners if enabled :p

Reply:
duhh I'm not in my best form today, sorry for the noise

Reply:
actually had that check in there before but my commit got lost. anyway got that fixed now